src/app/feed/feed-header/feed-header.component.html

```diff
@@ -9,33 +9,30 @@
 			<div class="input-group">
 				<label for="search">Search On Loklak.</label>
 
-				<input type="text" class="form-control" id="search" name="search" required
-								[formControl]="searchInputControl" placeholder="I'm feeling awesome!" (keyup)="gotolink($event)" [(ngModel)]=query (click)="visibility=true"/>
+				<input mdInput required type="text" id="search" name="search" placeholder="I'm feeling awesome!"
+								[formControl]="searchInputControl" [mdAutocomplete]="searchSuggestBox"
+								(keyup.enter)="relocateEvent.emit(query)" [(ngModel)]="query"/>
 
+				<md-autocomplete #searchSuggestBox="mdAutocomplete">
+					<md-option *ngFor="let suggestion of suggestionList" [value]="suggestion.query">{{suggestion.query}}</md-option>
+				</md-autocomplete>
 
 				<span class="input-group-btn">
 					<button class="btn" role="button" type="button" tabindex="-1"
-									aria-roledescription="Search the web using loklak search." (click)="searchEvantEmitter.emit()">
+									aria-roledescription="Search the web using loklak search." (click)="relocateEvent.emit(query)">
 						<i class="material-icons material-icons.md-light">search</i>
 					</button>
 				</span>
 			</div>
 		</div>
-		<div class="container suggesstion-box" #suggestionbox *ngIf="visibility && suggesstionList.length > 0">
```

Comment:
Hey @hemantjadon.

- [ ] I was saying about the scss part. Check it out in scss file.
- [ ]  limit suggestions to four. 
- [ ] Set autocomplete off. Because autocomplete from browser history is showing up still
- [ ] There is a gap between suggestion box and search input box . Make it attached to the search input.

Rest everything is good. :)